apps/dynatrace/perftest/00/dynatrace-operator.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: dynatrace
+spec:
+  values:
```

Comment:
I would have this as a base for perftest, rather than duplication to both perftest/00perftest /01